src/legacy/core_plugins/kibana/public/discover/components/field_chooser/_field_chooser.scss

```diff
@@ -8,4 +8,8 @@
 */
 .dscProgressBarTooltip__anchor {
   display: block;
-}
\ No newline at end of file
+}
+
+.toggleFieldFilterButton {
```

Comment:
Following our BEM naming scheme, I believe this should this also have the `.dsc` prefix.

Reply:
thank you! changed!